src/asciidoctor-core-api.js

```diff
@@ -177,15 +184,15 @@ Opal.Asciidoctor.AbstractNode.$$proto.setOption = function (name) {
   return this.$set_option(name);
 };
 
-Opal.Asciidoctor.AbstractNode.$$proto.getIconURI = function (name) {
+Opal.Asciidoctor.AbstractNode.$$proto.getIconUri = function (name) {
```

Comment:
I'm trying to align with AsciidoctorJ here. See http://static.javadoc.io/org.asciidoctor/asciidoctorj/1.6.0-alpha.3/org/asciidoctor/ast/ContentNode.html